tools/preprocess_emilia.py

```diff
@@ -71,6 +65,72 @@ def get_args():
     return parser.parse_args()
 
 
+def tokenize_by_CJK_char(text: str) -> str:
```

Comment:
I think it's better to put this and the following util functions in somewhere like utils.py , it's not used only for emilia dataset.